packages/api/src/schema/schema.yaml

```diff
@@ -2572,6 +2572,70 @@ components:
           type: number
           example: 200000
 
+    create-room-response:
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+
+    room:
+      type: object
+      table: rooms
+      required:
+        - id
+        - participants
+        - events
+      properties:
+        id:
+          type: string
+          unique: true
+          index: true
```

Comment:
Actually the `id` field is the only exception! We don't need any index here.
```suggestion
```

In the database, we have a separate column for the ID which is also the table primary key, and when querying by it we always check that table instead of the `id` in the JSON column. The index here would be redundant/unused.